File: lib/src/features/tripForm/models/trip.dart
class Trip {
  final String destination;
  final DateTime startDate;
  final DateTime endDate;
  final String transportation;
  final bool accommodation;
  final bool activities;

  Trip({
    required this.destination,
    required this.startDate,
    required this.endDate,
    required this.transportation,
    required this.accommodation,
    required this.activities,
  });
}

File: back/functions/src/domain/models/trip/trip.ts
export interface ITrip {
  id: string;
  name: string;
  startDate: string;
  endDate: string;
  origin: string;
  destination: string;
  transportation: string;
  createdAt: string;
  hotel: string;
  travelStyle: string;
  ladiesOnlyMetro: boolean;
  ladiesOnlyTaxi: boolean;
  loudNoiseSensitive: boolean;
  crowdFear: boolean;
  noIsolatedPlaces: boolean;
  lowCrime: boolean;
  publicTransportOnly: boolean;
  placesToVisit: string[];
}
